### PR TITLE
Bump Sentry Cocoa SDK to version 8.21.0

### PR DESCRIPTION
**Pull request by lucas-zimerman**
This PR will bump Sentry Cocoa to version 8.21.0.
No break changes for the end user has been found.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- build(ios): Bump `sentry-cocoa` to 8.21.0 ([#317](https://github.com/getsentry/sentry-cordova/pull/337))
```
Comment (lucas-zimerman) [inline CHANGELOG.md:7]:
```suggestion
- build(ios): Bump `sentry-cocoa` to 8.21.0 ([#337](https://github.com/getsentry/sentry-cordova/pull/337))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- build(ios): Bump `sentry-cocoa` to 8.21.0 ([#337](https://github.com/getsentry/sentry-cordova/pull/337))
+  - [changelog](https://github.com/getsentry/sentry-cocoa/releases/tag/8.21.0)
+  - [diff](https://github.com/getsentry/sentry-cocoa/compare/8.13.1...8.21.0)
```
Comment (krystofwoldrich) [inline CHANGELOG.md:9]: M: Missing new line before the next headline.

```suggestion
  - [diff](https://github.com/getsentry/sentry-cocoa/compare/8.13.1...8.21.0)

```

File: sample/package.json [hunk 3]
```diff
@@ -47,4 +47,4 @@
     "cordova": "^12.0.0",
     "typescript": "^5.2.2"
   }
-}
+}
```
Comment (krystofwoldrich) [inline sample/package.json:50]: L: Nit, missing a new line at the end of the file.

**Review by krystofwoldrich — APPROVED** [review]
LGTM 🚀 Thanks.

File: sample/package.json (hunk 3 above)
Comment (lucas-zimerman) [inline sample/package.json:50]:
```suggestion
}

```

**Pull request merged**